recipes/salad/stir-fry-cabbage.md

```diff
@@ -0,0 +1,58 @@
+# Stir-Fry Cabbage
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-29%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-0.5hr-red.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-VeryCheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+* **First cooked by:** [@heathercoraje](https://github.com/Heathercoraje)
+* **Date:** Week 11, 13 FACN2
+* **Author of Recipe**: [@heathercoraje](https://github.com/Heathercoraje)
+
+**The story behind this dish**
+The author of recipe often suffers from acid reflux and indigestion. Cabbage is known to help reduce stomach pain and intestinal Ulcers.
+[Read more](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1643665/). Because fresh cabbage juice isn't her favorite beverage, she came up with a brilliant idea to eat a large amount of cabbage and actually enjoy eating it.
+
+```
+return Stir-Fry Cabbage
+```
+
+## Ingredients
+
+A wok, vegetable oil (olive oil x), 1/2 of a large white cabbage, 3 pieces of 50g of slim Scallion (Green onion), 2 pieces of large dried chili pepper, a base sauce, 5 or more cloves of garlic.
+
+* Base sauce needs to be prepared boiled with following ingredients
+  * 4:5:1 ratio of Soy sauce, Oyster sauce and sugar.
+  * 5:5 ratio of Soy sauce and sugar for veggie friends.
+
+The bottom line of the dish is _timing_. It is essential to have all ingredients chopped and ready.
+
+### Ingredient preparation
+
+* Slice cabbage for width of 1cm or less
+* Chop green onions for width of less than 1cm
+* Chop chili pepper large enough so that people don't eat it accidentally
+* Slice garlic
+
+### Preparation
+
+1. Heat the wok and add enough vegetable oil. This is stir-fry dish. Make sure to be generous with oil
+
+2. Once the wok is hot enough (on medium low), add green onions and wait until the oil starts to smell green onion-y. This is to add a primary flavor to oil. (For 3-4 min)
+
+3. Add red chili pepper and stir. This is to add secondary flavor to oil (3min)
```

Comment:
flavor -> flavour

Reply:
no it's flavor you are a colonial you are not trust worthy